lms/templates/courseware/progress.html

```diff
@@ -140,9 +141,10 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
                             </div>
                         </div>
                     %endfor
+                %endfor
                  </div>
-                <button class="detail-collapse" aria-live="polite"><i class="fa fa-caret-down"></i>
-                    <span class="requirement-detail">${_("More")}</span><span class="sr"> detail</span>
+                <button class="detail-collapse" aria-live="polite"><i class="fa fa-caret-up"></i>
+                    <span class="requirement-detail">${_("Less")}</span><span class="sr"> detail</span>
```

Comment:
same comment here about translation - it's best if the translator can work with a single sentence or phrase.  Also, "detail" isn't being translated at all here.